source/blog/2018-01-16-ovirt-report-using-grafana.html.md

```diff
@@ -25,7 +27,46 @@ You will need to add a [PostgreSQL data source](http://docs.grafana.org/features
 For example:
 ![](/images/grafana_data_source_example.png)
 
-You may want to add a read only user to connect the history database - [Allowing read only access to the history database](https://www.ovirt.org/documentation/data-warehouse/Allowing_Read_Only_Access_to_the_History_Database/)
+You may want to add a read only user to connect the history database :
+
+**Note:** In oVirt 4.2 we ship postgres 9.5 through the Software Collection. 
+1. In order to run psql you will need to run:
+     
+       # su - postgres 
+       # scl enable rh-postgresql95 -- psql ovirt_engine_history
+**Allowing Read-Only Access to the History Database**
+2. Create the user to be granted read-only access to the history database:
+
+       # CREATE ROLE [user name] WITH LOGIN ENCRYPTED PASSWORD '[password]';
```

Comment:
put a different prefix for database shell and bash

bash root -> #
database shell -> ~ (or something other than #)

thus making this more explicit